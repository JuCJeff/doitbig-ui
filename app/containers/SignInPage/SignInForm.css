input, textarea, select, button {
	font-size: 16px;
	font-family: Helvetica, Arial, sans-serif;
	outline: none;
	margin: 0px;
	padding: 0px;
}

.title_h1 {
	font-size: 40px;
	text-align: center;
	font-weight: bold;
	letter-spacing: 4px;
}

.title_h4 {
	font-size: 16px;
	text-align: center;
	margin-top: 35px;
	color: #999;
	font-weight: normal;
}

.signinRow {
	width: 100%;
	height: 35px;
	margin-top: 40px;
	border: 0;
	box-shadow: none;
	border-bottom: 1px solid #ddd;
	margin-bottom: 20px;
}

.upBtns {
	margin-top: 50px;
	margin-left: 145px;
}

.upBtns button {
	line-height: 50px;
	height: 50px;
	background-color: #42425c;
	color: #fff;
	padding: 0 60px;
	border: none;
	border-radius: 5px;
}

.agreeView {
	text-decoration: underline;
	margin-top: 35px;
	text-align: center;
}

@media screen and (max-width: 1230px) {
    .contentView {width: 1230px};
    .container {width: 100%};
}


/* body {
	margin: 0;
	padding: 0;
	font-family: sans-serif;
}
.box {
	position: absolute;
	top: 50%;
    left: 75%;
	transform: translate(-50%, -50%);
	width: 500px;
	padding: 100px;
	background: rgba(0,0,0,.8);
	box-sizing: border-box;
	box-shadow: 0 15px 25px rgba(0,0,0,.5);
	border-radius: 10px;
}

.box h2 {
	margin: 0 0 30px;
	padding: 0;
	color: #FFF;
	text-align: center;
}
.box .inputBox {
	position: relative;
}
.box .inputBox input {
	width: 100%;
	padding: 10px 0;
	font-size: 16px;
	color: #FFF;
	letter-spacing: 1px; 
	margin-bottom: 30px;
	border: none;
	border-bottom: 1px solid #FFF;
	outline: none;
	background: transparent;
}
.box .inputBox label {
	position: absolute;
	top: 0;
	left: 0;
	letter-spacing: 1px;
	padding: 10px 0;
	font-size: 16px;
	color: #FFF;
	pointer-events: none;
	transition: .5s;
}
.box .inputBox input:focus ~ label,
.box .inputBox input:valid ~ label {
	top: -18px;
	left: 0;
	color: #FF8A65;
	font-size: 12px;
}
.box input[type="submit"] {
	background: transparent;
	border: none;
	outline: none;
	color: #FFF;
	background: #FF8A65;
	margin-top: 30px;
	padding: 10px 20px;
	cursor: pointer;
	border-radius: 5px;
} */