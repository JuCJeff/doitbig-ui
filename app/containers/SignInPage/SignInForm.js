import React from 'react';
import './SignInForm.css';

/* eslint-disable react/prefer-stateless-function */
export default class SignInForm extends React.PureComponent {
  render() {
    return (
      <div className="box">
        <h1 className="title_h1">STS STUDENT</h1>
        <h4 className="title_h4">Please sign in to your account.</h4>

        <form action="http://localhost:3001/login" method="get">
          <div className="signinRow">
            <input type="email" placeholder="Email" name="email" />
          </div>
          <div className="signinRow">
            <input type="password" placeholder="Password" name="password" />
          </div>
          <div className="upBtns text-center">
            <button type="submit" className="btn-default" href="/">
              Log in
            </button>
          </div>
          <div className="agreeView">
            <a href="/signup">Do not have an account? Sign up.</a>
          </div>
        </form>
      </div>
    );
  }
}
