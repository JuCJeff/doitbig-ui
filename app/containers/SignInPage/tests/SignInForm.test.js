import React from 'react';
import { shallow } from 'enzyme';

import SignInForm from '../index';

describe('<SingInForm />', () => {
  it('should render the page message', () => {
    const renderedComponent = shallow(<SignInForm />);
    expect(renderedComponent).toMatchSnapshot();
    // expect(
    //   renderedComponent.contains(
    //     <div className="body">
    //       <img alt="background" src="IMAGE_MOCK" />
    //       <SignInForm />
    //     </div>,
    //   ),
    // ).toEqual(true);
  });
});
