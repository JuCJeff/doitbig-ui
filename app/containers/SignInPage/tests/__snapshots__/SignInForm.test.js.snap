// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SingInForm /> should render the page message 1`] = `
<div
  className="body"
>
  <img
    alt="background"
    src="IMAGE_MOCK"
  />
  <SignInForm />
</div>
`;
